Facades/FinishCheckout/Services/ShopwareOrderUpdater.php

```diff
@@ -130,10 +128,9 @@ public function updateReferencesFromMollieOrder(Order $swOrder, \Mollie\Api\Reso
                     }
 
                     if (
-                        method_exists($detail->getAttribute(), 'getBasketItemId')
-                        && method_exists($detail->getAttribute(), 'setMollieTransactionId')
+                        method_exists($detail->getAttribute(), 'setMollieTransactionId')
                         && method_exists($detail->getAttribute(), 'setMollieOrderLineId')
-                        && (int)$detail->getAttribute()->getBasketItemId() === $transactionItem->getBasketItemId()
```

Comment:
i think i recently heard that problem from a merchant who updated systems.
could we create an OR condition, so that either the NEW or the OLD way lead to to this? just to be safe,
rest looks totally fine :) thanks

Reply:
It leads to weird behavior in my test shop. It always picks the shipping item. While matching SKUs works flawlessy in my testing.